src/amo/sagas/addonsByAuthors.js

```diff
@@ -18,13 +19,14 @@ export function* fetchAddonsByAuthors({ payload }) {
   yield put(errorHandler.createClearingAction());
 
   try {
+    const type = getAddonTypeFilter(addonType);
```

Comment:
We should probably stick to "addonTypeFilter" because it is a bit confusing to mix type, addonType, and addonTypeFilter.